ci-operator/config/shiftstack/ci/shiftstack-ci-release-4.16.yaml

```
@@ -209,6 +209,11 @@ tests:
   steps:
     cluster_profile: openstack-vh-mecha-az0
     workflow: openshift-e2e-openstack-perfscale-ingress-perf
+- as: e2e-openstack-network-perf
+  minimum_interval: 336h
+  steps:
+    cluster_profile: openstack-vh-mecha-az0
+    workflow: openshift-e2e-openstack-perfscale-network-perf
```

(reposting because my comment got lost in a rebase)

What do you think?

```suggestion
    workflow: openshift-e2e-openstack-ovn-perfscale-network
```